src/components/ContactUs/index.js

```diff
@@ -0,0 +1,194 @@
+                        <ErrorMsg>{errors.firstName}</ErrorMsg>
+                      )}
+                    </td>
+                    <td>
+                      <label htmlFor="lastName">Last Name:</label>
+                      <ContactInput
+                        type="text"
+                        id="lastName"
+                        placeholder="Doe"
+                        name="lastName"
+                        value={values.lastName}
+                        onBlur={handleBlur}
+                        onChange={handleChange}
+                      />
+                      {errors.lastName && touched.lastName && (
+                        <ErrorMsg>{errors.lastName}</ErrorMsg>
+                      )}
+                    </td>
+                  </tr>
+                  <tr>
+                    <td>
+                      <label htmlFor="phone">Phone:</label>
+                      <ContactInput
+                        type="text"
+                        id="phone"
+                        placeholder="(123) 123-4567"
+                        name="phone"
+                        onChange={handleChange}
+                        onBlur={handleBlur}
+                        value={values.phone}
+                      />
+                      {errors.phone && touched.phone && (
+                        <ErrorMsg>{errors.phone}</ErrorMsg>
+                      )}
+                    </td>
+                    <td>
+                      <label htmlFor="email">Email:</label>
+                      <ContactInput
+                        type="text"
+                        id="email"
+                        placeholder="jane-doe@example.com"
+                        name="email"
+                        value={values.email}
+                        onChange={handleChange}
+                        onBlur={handleBlur}
+                      />
+                      {errors.email && touched.email && (
+                        <ErrorMsg>{errors.email}</ErrorMsg>
+                      )}
+                    </td>
+                  </tr>
+                </tbody>
+              </ContactUsTable>
+              <label htmlFor="message">Message:</label>
+              <ContactTextArea
+                type="text"
+                id="message"
+                onChange={handleChange}
+                name="msg"
+                row="5"
+                value={values.msg}
+                onBlur={handleBlur}
+              />
+              {errors.msg && touched.msg && <ErrorMsg>{errors.msg}</ErrorMsg>}
+              <br />
+              <ContactUsBtn type="submit">Submit</ContactUsBtn>
+              <ContactCloseBtn onClick={props.click}>Exit</ContactCloseBtn>
+            </ContactForm>
+          </div>
+        )}
+      </ContactFormContainer>
+    </ContactBackground>
+  );
+};
+
+const EnhancedWithFormik = withFormik({
+  mapPropsToValues: () => ({
+    firstName: "",
```

Comment:
should we assign these defaults to a var ie: const initialValues = {...} and just pass in initialValues here?